bothan-api/server-cli/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "bothan-api-cli"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
Let's try add this. It should make binary name `bothan`
```
[[bin]]
name = "bothan"
````

Reply:
and not sure should we need other fields? Just paste hermes example https://github.com/informalsystems/hermes/blob/master/crates/relayer-cli/Cargo.toml

Reply:
I think any other fields are separate from this pr as it doesn't pertain to the src, will add in another pr if necessary